bool request

-----
float32[] x
float32[] y
float32[] theta
float32[] xt
float32[] yt
sensor_msgs/LaserScan[] all_laser